javaLoginCore/src/main/java/com/hamdan/forzenbook/java/login/core/di/JavaLoginModule.java

```diff
@@ -0,0 +1,92 @@
+package com.hamdan.forzenbook.java.login.core.di;
+
+import static com.hamdan.forzenbook.java.core.GlobalConstants.LOGIN_BASE_URL;
+
+import android.content.Context;
+
+import androidx.room.Room;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.hamdan.forzenbook.java.login.core.data.database.LoginDao;
+import com.hamdan.forzenbook.java.login.core.data.database.LoginDatabase;
+import com.hamdan.forzenbook.java.login.core.data.network.LoginService;
+import com.hamdan.forzenbook.java.login.core.data.repository.LoginRepository;
+import com.hamdan.forzenbook.java.login.core.data.repository.LoginRepositoryImpl;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginGetCredentialsFromNetworkUseCase;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginGetCredentialsFromNetworkUseCaseImpl;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginGetStoredCredentialsUseCase;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginGetStoredCredentialsUseCaseImpl;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginStringValidationUseCase;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginStringValidationUseCaseImpl;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginValidationUseCase;
+import com.hamdan.forzenbook.java.login.core.domain.usecase.LoginValidationUseCaseImpl;
+
+import javax.inject.Named;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.hilt.InstallIn;
+import dagger.hilt.android.components.ViewModelComponent;
+import dagger.hilt.android.qualifiers.ApplicationContext;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+@InstallIn(ViewModelComponent.class)
+@Module
+public class JavaLoginModule {
+    final private String MODULE_NAME = "java_login";
```

Comment:
Again, not a module, but a retrofit name. Maybe `javaLoginRetrofit`

Reply:
renamed